src/components/cell-tooltip/CellTooltipVertical.js

```diff
@@ -0,0 +1,44 @@
+import React, { useLayoutEffect, useRef } from 'react';
+
+export default function CellTooltipVertical(props) {
+  const {
+    hoveredCellInfo,
+    cellIndex,
+    numCells,
+    uuid,
+  } = props;
+  // Check that all data necessary to show the tooltip has been passed.
+  if (!hoveredCellInfo || !uuid || !cellIndex || !numCells) {
+    return null;
+  }
+  // If we're in the component that triggered the event, do not show the vertical line.
+  if (hoveredCellInfo.uuid === uuid) {
+    // In the future, potentially show a tooltip with `hoveredCellInfo.status`.
+    return null;
+  }
+  const y = 0;
+  const lineWidth = 1;
+  // Compute the x-position of the vertical emphasis element
+  // based on the width of the sibling canvas.
+  const el = useRef(null);
```

Comment:
I think the react docs often take this style, but I'd prefer a name like `ref` to make it clear this is not a plain element. (I've forgotten the `.current` sometimes, and been confused when normal DOM properties weren't available.)

Reply:
That is a good point. I have updated this.